src/FluentValidation.Tests/OnFailureTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FluentValidation.Tests
+{
+	using System.Diagnostics;

```

Comment:
Can we make these `using` statements consistent? Some are inside the namespace, and some are outside. The rest of the project tends to have them all inside the namespace declaration.

Reply:
Fixed